docs/apple_development.md

```diff
@@ -20,19 +20,6 @@ To open the iOS project on XCode:
   open ./platforms/ios/Outline.xcworkspace
 ```
 
-### Adding support for your device in XCode
```

Comment:
This issue is not specific to 13.2. Please restore (and remove the version number).